lib/tasks/generate_random_data.rake

```diff
@@ -0,0 +1,26 @@
+namespace :generate_random_data do
+
+  desc 'Generates a million random entries in users table'
```

Comment:
Can you make the number configurable, move it up into a constant and probably parse from command line args? 

Default can be 1M

Reply:
Yes sure, doing that ASAP.

Reply:
Done!